browser/src/canvas/CanvasSectionContainer.ts

```diff
@@ -1011,6 +957,37 @@ class CanvasSectionContainer {
 		}
 	}
 
+	public onNewDocumentTopLeft() {
```

Comment:
what it does? not obvious from the name, can we have comment above?

Reply:
Now we need to separate the view management from CanvasSectionContainer. Because multi-page view and possible future others need to manage the coordinates themselves.

Views need to set the document top left. In that case, CanvasSectionContainer needs to know when the document is scrolled. I think name is self-explaining but i will also add a comment there.